### PR TITLE
[wip] [test] Port ReFrame checks on Arolla and Tsa

### PR DESCRIPTION
**Pull request by lucamar**
I start the pull request as work in progress since the checks defined in `collectives_halo.py` currently do not work on Tsa and I have started investigating the cause of the failure.
 
Furthermore, few more checks still need to be adapted to run on Arolla and Tsa to complete the full set of checks that we run on the current MCH production system (Kesch).

### CODE REVIEW

File: cscs-checks/cuda/cuda_checks.py [hunk 1]
```diff
@@ -12,7 +12,6 @@ def __init__(self):
         if self.current_system.name == 'kesch':
```
Comment (kraushm) [inline cscs-checks/cuda/cuda_checks.py]: This set of tests still needs to be ported to Tsa/Arolla, or are they discarded?

**Comment by vkarak** [comment]
@lucamar Is this a WIP? Or ready to be reviewed?

**Comment by lucamar** [comment]
@vkarak Still WIP
> @lucamar Is this a WIP? Or ready to be reviewed?

**Comment by lucamar** [comment]
@jenkins-cscs retry none

**Comment by lucamar** [comment]
Since I have many changes in progress, I will close the pull request to avoid the CI run after every commit. I will re-open the pull request as soon as I complete the checks update.

**Pull request closed without merging**